control/controlclient/hostinfo_windows_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package controlclient
+
+import (
+	"os/exec"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestOsVersionWindows(t *testing.T) {
+	out, err := exec.Command("cmd", "/c", "ver").Output()
+	if err != nil {
+		t.Fatalf("`ver` error: %v", err)
+	}
+	rx := regexp.MustCompile(`\d+\.\d+\.\d+(\.\d+)?`)
```

Comment:
add parens around the first three numbers and then do an exact `==` match on that $1 part instead of HasPrefix

Reply:
done